@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -597,39 +592,16 @@ export class VisibleTileSet {
             const tiles = renderListEntry.visibleTiles;
             tiles.forEach(tile => {
                 minElevation = Math.min(minElevation, tile.minElevation);
-                maxElevation = Math.max(maxElevation, tile.maxElevation);
+                maxElevation = Math.max(maxElevation, tile.maxElevation + tile.maxGeometryHeight);
```

Comment:
This already contains a bug, see my comments above: maxElevation (getter) already sums up elevation and geometry height.

Reply:
fixed, `Tile.maxElevation` contains ground elevation